source/features/link-to-changelog-file.tsx

```diff
@@ -15,16 +15,13 @@ interface FileType {
 }
 
 const getCacheKey = (): string => `changelog:${getRepo()!.nameWithOwner}`;
-const changelogNames = new Set(['changelog', 'news', 'changes', 'history', 'release', 'whatsnew']);
 
+const regexChangelogMarkdownTextFiles = /^(changelog|news|changes|history|release|whatsnew)\.(mdx?|mkdn?|mdwn|mdown|markdown|litcoffee|txt|rst)$/i;
```

Comment:
There are changelogs with no extension (e.g. https://github.com/sphinx-doc/sphinx/blob/4.x/CHANGES)
```suggestion
const regexChangelogMarkdownTextFiles = /^(changelog|news|changes|history|release|whatsnew)(\.(mdx?|mkdn?|mdwn|mdown|markdown|litcoffee|txt|rst))?$/i;
```

Also the name could be shorter, it's not always Markdown and we know it's a regex by looking at it. Maybe just `changelogFiles`?

Reply:
> it's not always Markdown

You've read my mind! I was going to ask about files that have no extension 👍 

> Maybe just `changelogFiles`?

Yes, 👍